docs/reference/release-notes/860.md

```diff
@@ -113,6 +113,18 @@ On Camunda 8 SaaS, you can now configure hourly backup intervals. Simply head to
 
 In Operate, the visualization of the executed path of a process instance is improved. Both executed sequence flows and flow nodes are highlighted, allowing you to display how often each flow node was executed.
 
+### Write rate limiting based on exporting rate
+
+<!-- https://github.com/camunda/product-hub/issues/1426 -->
+
+We introduce a new unified flow control mechanism inside of Zeebe that is able to limit user commands (by default it tries to achieve 200ms response times) and rate limit writes of new records in general.
+
+Limiting the write rate is a new feature that can be used to prevent building up an excessive exporting backlog. There are two ways to limit the write rate, either by setting a static limit or by enabling throttling that dynamically adjust the write rate based on the exporting backlog and rate. If activated you might see more back pressure in cases where exporters are not exporting fast enough a backlog builds up. The back pressure error message will contain information to explain what is happening and you can read documentation to better understand what to do in such cases.
```

Comment:
Non-blocking suggestion (reword):

You can use this new feature to prevent the buildup of an excessive exporting backlog.

*  To limit the write rate, you can either set a static limit or enable throttling that dynamically adjusts the write rate based on the exporting backlog and rate.
* If activated, you might see more backpressure in cases where exporters are not exporting fast enough, so that a backlog builds up. The backpressure error message contains information to explain what is happening, and you can also refer to the documentation to understand what to do in such cases.

(bulleting helps break it up for easier scanning, 'you' addresses/engages the reader more, backpressure vs back pressure, active voice etc)